mk/distribution.mk

```diff
@@ -51,6 +51,9 @@ endif
 	find $$@ -type f | xargs chmod 555
 	# Text files don't have executable access
 	find $$@ -type f -exec grep -I -q '' {} \; -print | xargs chmod 444
+ifeq ($(1),windows)
+	find $$@/bin $(if $(findstring Darwin,$(shell uname)),-perm +111,-executable) -type f -exec mv {} {}.exe \;
```

Comment:
It's hard for me to grasp the full understanding of this line, but is the `Darwin` correct here?

Reply:
Haha :) windows is the target OS for the build (the GOOS). Darwin is the OS where we're building this. `find` on darwin doesn't support `-executable` and linux doesn't support `-perm`. I'm going to add a comment to make this clearer.

Reply:
Let me know if this is clearer with the new comments